apps/backend/src/models/Institution.ts

```diff
@@ -3,6 +3,6 @@ export class Institution {
     public id: number,
     public name: string,
     public country: number,
-    public verified: boolean
+    public rorId?: string
```

Comment:
Necessary optional?

Reply:
optional is necessary here, Institution can be without rorId